airbyte-server/src/main/java/io/airbyte/server/apis/DestinationOauthApiController.java

```diff
@@ -10,26 +10,34 @@
 import io.airbyte.api.model.generated.OAuthConsentRead;
 import io.airbyte.api.model.generated.SetInstancewideDestinationOauthParamsRequestBody;
 import io.airbyte.server.handlers.OAuthHandler;
+import io.micronaut.context.annotation.Context;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Post;
 import java.util.Map;
-import javax.ws.rs.Path;
-import lombok.AllArgsConstructor;
 
-@Path("/v1/destination_oauths")
-@AllArgsConstructor
+@Controller("/api/v1/destination_oauths")
+@Context
```

Comment:
Is `@Context` necessary here?  Per the [docs](https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Context.html) `WARNING: This annotation should be used sparingly as Micronaut is designed in such a way as to encourage minimal bean creation during startup.`

Reply:
@colesnodgrass While testing, I have observed that the first call to the API are longer than the others because of bean initialization. I made the bean to have an eager initialization.